Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.cpp

```diff
@@ -20,6 +20,13 @@ namespace Multiplayer
 {
     constexpr float defaultConnectionMessageFontSize = 0.7f;
     const AZ::Vector2 viewportConnectionBottomRightBorderPadding(-40.0f, -40.0f);
+    const char* centerViewportDebugTitle = "Multiplayer Editor";
+    const char* clientStatusTitle = "Multiplayer Client Status:";
+    const char* onServerLaunchedMessage = "(1/3) Launching server...";
+    const char* onServerLaunchFailMessage = "(1/3) Could not launch editor server.\nSee console for more info.";
+    const char* onEditorConnectionAttemptMessage = "(2/3) Editor tcp connection attempt #%i.";
+    const char* OnEditorSendingLevelDataMessage = "(3/3) Editor is sending the editor-server the level data packet.";
+    const char* onConnectToSimulationFailMessage = "EditorServerReady packet was received, but connecting to the editor-server's network simulation failed! Is the editor and server using the same sv_port (%i)?";
```

Comment:
You should either be doing constexpr char[] or constexpr AZStd::string_view here. I'm not actually sure how well string_view would play with AZStd::string::format though.

Reply:
roger! i see :D fixed!